.gitignore

```diff
@@ -1,10 +1,69 @@
-.idea/
-project/project/target
-project/project/project
-project/target/
-target/
-deploy_key
+### Intellij ###
+
+.idea
+out/
+
+### Java ###
+
+*.class
+*.log
+
+### macOS ###
+
 .DS_Store
-output/*
-secret_key.asc
-/shading/
+
+### SBT ###
+
+dist/*
+target/
+lib_managed/
+src_managed/
+project/boot/
+project/plugins/project/
+.history
+.cache
+.lib/
+
+### Scala ###
+
+*.metals
+
+### Vim ###
+
+# Swap
+[._]*.s[a-v][a-z]
+[._]*.sw[a-p]
+[._]s[a-rt-v][a-z]
+[._]ss[a-gi-z]
+[._]sw[a-p]
+
+# Session
+Session.vim
+Sessionx.vim
+
+# Temporary
+.netrwhist
+
+# Auto-generated tag files
+tags
+
+# Persistent undo
+[._]*.un~
+
+# Coc configuration directory
+.vim
+
+### VisualStudioCode ###
+
+.vscode/*
+!.vscode/settings.json
+!.vscode/tasks.json
+!.vscode/launch.json
+!.vscode/extensions.json
+
+### sbt-fix ###
+# Scalafix & Scalafmt configuration files are automatically downloaded by `sbt-fix`
+# https://github.com/47degrees/sbt-fix
+
+.scalafix.conf
+.scalafmt.conf
```

Comment:
should we remove it?

Reply:
It seems these files are managed by sbt-fix so we wouldn't need to keep them updated in the repo.